sql/src/main/java/io/crate/operation/udf/UserDefinedFunctionService.java

```diff
@@ -60,12 +60,23 @@
 
     private final ClusterService clusterService;
     private final Functions functions;
+    private static Map<String, UDFLanguage> languageRegistry;
+
+
+    public static UDFLanguage getLanguage(String languageName) {
```

Comment:
annotate with `@Nullable` or throw InvalidArgumentException (or something that indicates that language does not exist)